app/assets/javascripts/discourse/app/components/topic-list.js

```diff
@@ -190,4 +190,24 @@ export default Component.extend(LoadMore, {
       this.rerender();
     });
   },
+
+  keyDown(e) {
+    if (e.keyCode === 13 || e.keyCode === 32) {
+      // enter or space
+      let self = this;
+
+      let onKeyDown = (sel, callback) => {
+        let target = $(e.target).closest(sel);
```

Comment:
can we use native js here ? https://developer.mozilla.org/fr/docs/Web/API/Element/closest

Reply:
it's possible behavior is slightly different